pro/src/components/pages/Offers/Offer/Stocks/EventStocks.jsx

```diff
@@ -194,7 +194,36 @@ const EventStocks = ({ offer, reloadOffer, isCompletingDraft }) => {
     })
   }
 
-  const submitDraft = e => submitStocks(e, true)
+  const submitDraft = e => {
+    const isEmptyStocks = stocks => {
+      if (stocks.length > 1) {
+        return false
+      }
+      const {
+        price,
+        quantity,
+        beginningTime,
+        bookingLimitDatetime,
+        beginningDate,
+      } = stocks[0]
```

Comment:
tu as un risque que ca plante avec un erreur genre `cannot read price of undefined` si jamais stocks est une liste vide

Reply:
C'est un peu particulier sur cette page. Il faut cliquer sur un bouton pour avoir accès au formulaire 😬 . Du coup au pire on a une liste avec un objet initialisé vide. 
Ca va bientôt changer !